Artifacts.toml

```diff
@@ -0,0 +1,7 @@
+[gap_hulpke_extraperfect]
+git-tree-sha1 = "9ef6636e5ca29fea6e6513aec2c977ab5c1b89cd"
+lazy = true
+
+    [[gap_hulpke_extraperfect.download]]
+    sha256 = "2190f2d128f55194b7afc905fef7d77e940a196c102fb3ecdd456813a4f62670"
+    url = "https://github.com/hulpke/extraperfect/archive/80c0cf27bf98ba3eabd287db5ab4fd0f249e0559.tar.gz"
```

Comment:
Ok. So I would change this here using a fixed release asset name for the v0.12.2 release, provide a slightly modified tarball (I would deflate it one level) to someone with write access and then we do a v0.12.2 shortly after merging this?
Or would it be easier to add the tarball to the v0.12.1 release so that we do not break master between merging this and the release?.

Reply:
> Sorry, I totally missed that this is not your repository. We could indeed put it in some Oscar release asset but this seems somewhat strange.

This is how Hecke does it, see e.g. https://github.com/thofma/Hecke.jl/releases/tag/v0.13.4
Thus I proposed to do it similarly.

Reply:
Ok, so in Hecke, the assets just got added to the newest release as of the time of merge. So in our case, that would be Oscar.jl v0.12.1.
I will prepare the tarball, send it to @fingolfin to put in the v0.12.1 release, and then update this PR accordingly..

Reply:
> I see, ok, fine with me then, just give it a reasonable name.

After thinking about it for a bit: The https://github.com/hulpke/extraperfect repo does not have any license. By providing a tarball as a release asset in the Oscar repo, we would redistribute some unlicensed code. I am not that firm with copyright law, but that still seems a bit whacky to me.
Giving some artifact URL on hulpke's namespace seems a lot less weird and risky to me. Maybe I/me/someone/... should reach out to hulpke (there is a mail address on his github profile) and ask him if he could make a release, or put a license in the repo, or ....
(no @-mentioning him by on purpose)

Reply:
Ah, I missed this bit about mirroring. OK, I'll look into that as well.

Reply:
OK, I uploaded https://github.com/oscar-system/GAP.jl/releases/download/archive-tag/extraperfect.tar.gz

I created it like this, for maximum reproducibility:
```
#!/bin/sh
export SOURCE_DATE_EPOCH=$(git log -1 --pretty=%ct)
rm -rf extraperfect
mkdir extraperfect
gzip -9 -n -c perf27.grp > extraperfect/perf27.grp.gz
gzip -9 -n -c perf33.grp > extraperfect/perf33.grp.gz
gtar --mtime="$(git log -1 --pretty=%cI)" --owner=0 --group=0 --numeric-owner -cvf extraperfect.tar extraperfect
gzip -n -c extraperfect.tar > extraperfect.tar.gz
```
Note that I compressed the two `.grp` files on purpose -- this saves a little bit of storage for the user, and is handled transparently by GAP.